x-pack/platform/plugins/shared/stack_connectors/server/connector_types/openai/openai.ts

```diff
@@ -143,105 +197,141 @@ export class OpenAIConnector extends SubActionConnector<Config, Secrets> {
   }
 
   protected getResponseErrorMessage(error: AxiosError<{ error?: { message?: string } }>): string {
-    // handle known Azure error from early release, we can probably get rid of this eventually
```

Comment:
please don't delete these comments

Reply:
don't resolve a comment until you fulfill its requirements. This cannot be deleted